pkgs/by-name/lc/lcov/package.nix

```diff
@@ -58,6 +59,12 @@ stdenv.mkDerivation (finalAttrs: {
     done
   '';
 
+  doInstallCheck = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
+
+  versionCheckProgramArg = "--version";
```

Comment:
Does it work without this line? It seem the hook tries `--version` automatically (after --help).